src/EvolvableLinkTrait.php

```diff
@@ -19,13 +19,13 @@ trait EvolvableLinkTrait
      *
      * @return EvolvableLinkInterface
```

Comment:
Do we want to remove explicit `@return` tags, given the return types are now code-explicit?  Removing tags when code adds explicitness is what I usually see from the doc/stat-anal tools.

Reply:
Yeah, the docblocks are stolen from the interface in most cases so I just did what those did.